deploy/crds/managed_v1alpha1_subjectpermission_cr.yaml

```diff
@@ -3,5 +3,10 @@ kind: SubjectPermission
 metadata:
   name: example-subjectpermission
 spec:
-  # Add fields here
-  size: 3
+  subjectKind: Group
+  subjectName: Test
+  permissions:
+    -
+      clusterRoleName: bar
```

Comment:
@fahlmant @sam-nguyen7 I think this should be `roleName` which means the API will need to be changed.

Reply:
@jharrington22 No this is correct. This is creating a rolebinding for a service account (namespaced) to a clusterRole. This is in the spec. I can explain why its a clusterRole not a role

Reply:
Right, would we have use a rolebinding in some cases?

Reply:
As of right now, no the design is only for ClusterRoles. Personally, I think we should have support for both, but as far as the spec goes, this operator only makes bindings to ClusterRoles.